src/platform/packages/shared/response-ops/rule_params/synthetics_monitor_status/v1.ts

```diff
@@ -63,6 +63,7 @@ export const syntheticsMonitorStatusRuleParamsSchema = schema.object(
     tags: schema.maybe(schema.arrayOf(schema.string())),
     monitorTypes: schema.maybe(schema.arrayOf(schema.string())),
     projects: schema.maybe(schema.arrayOf(schema.string())),
+    investigation_guide: schema.maybe(schema.string()),
```

Comment:
@justinkambic Why is investigation_guide added here? Do I need to do the same for dashboards?

Reply:
No, in fact I do not think we need this here _at all_, well-spotted.

Reply:
fbf5bf52b12d2